components/centraldashboard/app/api.ts

```diff
@@ -5,22 +5,22 @@ import {KubernetesService} from './k8s_service';
 
 /** API routes exposing information from Kubernetes */
 export function api(kubeConfig: KubeConfig): express.Router {
-  const k8sService = new KubernetesService(kubeConfig);
-  return express.Router()
-      .get(
-          '/platform-info',
-          async (_: express.Request, res: express.Response) => {
-            res.json(await k8sService.getPlatformInfo());
-          })
-      .get(
-          '/namespaces',
-          async (_: express.Request, res: express.Response) => {
-            res.json(await k8sService.getNamespaces());
-          })
-      .get(
-          '/activities/:namespace',
-          async (req: express.Request, res: express.Response) => {
-            res.json(
-                await k8sService.getEventsForNamespace(req.params.namespace));
-          });
+	const k8sService = new KubernetesService(kubeConfig);
```

Comment:
Can we just revert this file out for now? It looks like the tabs are even more out of whack.